app/va/va_profile/va_profile_client.py

```diff
@@ -48,6 +53,62 @@ def init_app(
         self.ssl_key_path = ssl_key_path
         self.statsd_client = statsd_client
 
+    def get_profile(self, va_profile_id: 'RecipientIdentifier') -> 'Profile':
+        recipient_id = transform_to_fhir_format(va_profile_id)
+        oid = OIDS.get(IdentifierType.VA_PROFILE_ID)
+        url = f'{self.va_profile_url}/profile-service/profile/v3/{oid}/{recipient_id}'
+        data = {'bios': [{'bioPath': 'contactInformation'}, {'bioPath': 'communicationPermissions'}]}
+
+        try:
+            response = requests.post(url, json=data, cert=(self.ssl_cert_path, self.ssl_key_path), timeout=(3.05, 1))
+            response.raise_for_status()
+        except (requests.HTTPError, requests.RequestException, requests.Timeout) as e:
+            self._handle_exceptions(va_profile_id.id_value, e)
+        else:
+            response_json: Dict = response.json()
+            return response_json.get('profile', {})
+
+    def get_telephone_from_profile_v3(self, va_profile_id: 'RecipientIdentifier') -> str:
+        contact_info: 'ContactInformation' = self.get_profile(va_profile_id).get('contactInformation', {})
+        self.logger.info(f'V3 Profile - Retrieved ContactInformation: {contact_info}')
+
+        telephones: List['Telephone'] = contact_info.get(self.PHONE_BIO_TYPE, [])
+        phone_numbers = ', '.join([tel['phoneNumber'] for tel in telephones])
+        self.logger.info(f'V3 Profile telephones: {phone_numbers}' ')}')
+        mobile_telephones = [phone for phone in telephones if phone['phoneType'] == PhoneNumberType.MOBILE.value]
+        sorted_telephones = sorted(
+            mobile_telephones,
+            key=lambda phone: iso8601.parse_date(phone['createDate']),
+            reverse=True,
+        )
+        if sorted_telephones:
+            if (
+                sorted_telephones[0].get('countryCode')
+                and sorted_telephones[0].get('areaCode')
+                and sorted_telephones[0].get('phoneNumber')
+            ):
+                self.statsd_client.incr('clients.va-profile.get-telephone.success')
+            return (
+                '+'
+                + sorted_telephones[0]['countryCode']
+                + sorted_telephones[0]['areaCode']
+                + sorted_telephones[0]['phoneNumber']
+            )
```

Comment:
"mobile_telephones" isn't used again, so you might as well sort in place.
```suggestion
        mobile_telephones = [phone for phone in telephones if phone['phoneType'] == PhoneNumberType.MOBILE.value]
        if mobile_telephones:
            mobile_telephones.sort(
                key=lambda phone: iso8601.parse_date(phone['createDate']),
                reverse=True,
            )

            if (
                mobile_telephones[0].get('countryCode')
                and mobile_telephones[0].get('areaCode')
                and mobile_telephones[0].get('phoneNumber')
            ):
                self.statsd_client.incr('clients.va-profile.get-telephone.success')

            return (
                '+'
                + mobile_telephones[0]['countryCode']
                + mobile_telephones[0]['areaCode']
                + mobile_telephones[0]['phoneNumber']
            )
```

Reply:
I prefer verbosity but I understand that's not everybody's style.  Fixed.